CHANGELOG.md

```diff
@@ -490,7 +490,8 @@ See the [Tendermint v0.34.7 SDK changelog](https://github.com/tendermint/tenderm
 
 <!-- Release links -->
 
-[Unreleased]: https://github.com/cosmos/gaia/compare/v9.0.1...HEAD
```

Comment:
Do we still need to leave Unreleased here?

It seems to me that we do.

Reply:
@Marius did I get you ur comment: https://github.com/cosmos/gaia/pull/2418#discussion_r1171505326 right?

Reply:
That link is broken for main, it make sense only for `release/*` branches